src/main/scala/org/camunda/feel/impl/builtin/TemporalBuiltinFunctions.scala

```diff
@@ -98,4 +94,73 @@ class TemporalBuiltinFunctions(clock: FeelEngineClock) {
     ValDate(date.`with`(TemporalAdjusters.lastDayOfMonth()))
   }
 
+  private def timestampFunction = builtinFunction(
+    params = List.empty,
+    invoke = { case _ =>
+      val timestamp = clock.getCurrentTime.toEpochSecond
+      ValNumber(timestamp)
+    }
+  )
+
+  private def timestampFunction2 = builtinFunction(
+    params = List("datetime"),
+    invoke = {
+      case List(ValDateTime(datetime))        =>
+        val timestamp = datetime.toEpochSecond
+        ValNumber(timestamp)
+      case List(ValLocalDateTime(datetime))   =>
+        val timestamp = datetime.toInstant(ZoneOffset.UTC).getEpochSecond
+        ValNumber(timestamp)
+      case _                                  => ValNull
+
+    }
+  )
+
+  private def timestampFunction3 = builtinFunction(
+    params = List("timestamp","zoneId"),
+    invoke = {
+
+      case List(ValString(timestamp),ValString(zoneId))    =>
```

Comment:
:thought_balloon: The `timestamp` parameter should be a number and not a string.

`from unix timestamp` should be the reverse function of `to unix timestamp`.

```
from unix timestamp(to unix timestamp(@"2023-06-09T16:18:41Z"))
// "2023-06-09T16:18:41Z"
```

Reply:
If use `ValNumber(1596198450123)` will get the error of  `Integer literal is out of range for type Int`.

<img width="1592" height="175" alt="image" src="https://github.com/user-attachments/assets/5839e67f-df78-4eb2-916a-defd33562981" />